storage-proofs-post/tests/halo2_circuit.rs

```diff
@@ -118,30 +138,56 @@ where
 
 #[test]
 fn test_winning_post_circuit_2kib_halo2() {
+    init_logger();
```

Comment:
Maybe move logger initialization to the top of the inner test function so logging can be disabled via a single comment?